### ISSUE TITLE
StringDrop::seqs messages

### ISSUE THREAD

**Issue by p-pavel** [kenkangxgwe/lsp-wl#43]
lots of messages like this are generated during textDocument/publishDiagnostics:

```mathematica
StringDrop::seqs: 
   Sequence specification (+n, -n, {+n}, {-n}, {m, n}, or {m, n, s}) expected
     at position 2 in StringDrop[pollSockets[timeout_Integer] , 
     WolframLanguageServer`TextDocument`Private`startCharacter].
```

**Comment by kenkangxgwe** [comment]
The latest commit (https://github.com/kenkangxgwe/lsp-wl/commit/27f274b87128ab3cc902e0199c4fbd1ae53d9d1e) may fix this problem. The top of `develop` is not tested actually, several commits are pushed yesterday and really experimental. So bugs are not surprising. Thanks for the feedback but sorry that you have been the white rat.

**Comment by p-pavel** [comment]
> The latest commit ([27f274b](https://github.com/kenkangxgwe/lsp-wl/commit/27f274b87128ab3cc902e0199c4fbd1ae53d9d1e)) may fix this problem. The top of `develop` is not tested actually, several commits are pushed yesterday and really experimental. So bugs are not surprising. Thanks for the feedback but sorry that you have been the white rat.

It's ok as I switched to develop branch as it has all the activity and master branch does not work for me either (like StringJoin[Keys::usage<> expected to be a string etc).

**Comment by p-pavel** [comment]
Indeed the issue seem to be fixed in 27f274b87128ab3cc902e0199c4fbd1ae53d9d1e

**Comment by kenkangxgwe** [comment]
`Keys::usage` seems not there, but is your Mathematica an English version?
